### DISCUSSION TITLE
Let’s discuss the web we want

### DISCUSSION THREAD

**Discussion by aarongustafson** [WebWeWant/webwewant.fyi#72] [Announcements]
We’re are using Discussions as a place to talk about Web We Want submissions. Each discussion thread will be focused on a specific "want" from the website.

## Topics for discussion

**In joining this discussion, you must abide by [our Code of Conduct](https://github.com/WebWeWant/webwewant.fyi/blob/master/CODE_OF_CONDUCT.md).**

Good points of discussion include:

* clarifying questions (which can help us strengthen the want),
* alternative considerations,
* impacts addressing the want may have on other areas, and
* solutions you’ve seen in the community or from specific vendors

The purpose of these discussions is to determine how important the want is to folks, how it could be clarified or strengthened, etc. Constructive criticism is welcome, but non-constructive comments will be deleted.

## New discussion threads

We will be working to assign a unique discussion thread for each want, but we are working backwards so it may take a bit. If you want to open a thread for a want that has not been discussed yet (please search first), please title it with the want title and provide a link in the content of the initial post. Then add your discussion as the first comment.

**Comment by theojt** [comment]
I want improved safety in the Shopping experience.  I visited a website to purchase a tool.  Edge presented an alternative that directed me to a different website with the same tool at a greatly reduced price - almost too good to be true.  Before ordering, I reviewed that website (starts with an "A" and ends with "CME") and found a preponderance of negative reviews related to not only customer services but blatant rip-offs of consumers.

It would be helpful if the recommendations presented were from "vetted" websites (maybe validated via TrustPilot or such) to protect the viewer/consumer better.
